src/Bridges/Variable/bridges/hermitian.jl

```diff
@@ -0,0 +1,350 @@
+# Copyright (c) 2017: Miles Lubin and contributors
+# Copyright (c) 2017: Google Inc.
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+const EQ{T} = MOI.ConstraintIndex{MOI.ScalarAffineFunction{T},MOI.EqualTo{T}}
```

Comment:
Please don't re-introduce things like this. It just makes everything harder to read. Verbosity is okay.